generator/sbpg/targets/rust.py

```diff
@@ -59,6 +59,12 @@
 
 BASE_TIME_MSGS = ["MSG_OBS", "MSG_OSR", "MSG_SSR"]
 
+CHECK_FLAGS = """
+if self.time_source().ok()? == TimeSource::None {
```

Comment:
Actually like this one more because it checks against a specific variant. So if a message gets added that has a `TimeSource` but no `None` state for some reason we'll get a compile error